services/RaceMonitor/RaceMonitor.php

```diff
@@ -21,12 +23,17 @@ final class RaceMonitor
 
     private $message;
     private $atrMaxNumber;
+    private $blacklistPeriod;
 
     public function __construct()
     {
         $this->email = config('races.email');
         $this->target = config('races.target-url');
         $this->atrMaxNumber = config('races.atr-max-number');
+        $this->blacklistPeriod = config('races.blacklist-period');
+
+        //remove the blacklisted races after curtain amount of time
+        $this->unblacklist();
```

Comment:
Just a note for future projects - code which "actually performs actions" (like this) shouldn't go in the constructor. We try to use the constructor for just "setting up" the class.

I feel the code for un-blacklisting would be better suited to a scheduled command :)

Reply:
Thank you for the hint @craigsansam , I've decided to apply the changes. The app looks better now :)))